src/catchup/simulation/TxSimApplyTransactionsWork.cpp

```diff
@@ -555,9 +558,21 @@ TxSimApplyTransactionsWork::onRun()
         {
             return mApplyLedgerWork->getState();
         }
-        else if (mVerifyResults)
+        else
         {
-            checkResults(mApp, header.ledgerSeq, mResults);
+            if (mVerifyResults)
+            {
+                checkResults(mApp, header.ledgerSeq, mResults);
+            }
+            auto& closed =
+                mApp.getMetrics().NewTimer({"ledger", "ledger", "close"});
```

Comment:
Why not use a condition on lcl? We have the starting ledger, so we know where it's going to be after 100 ledgers.
This would be less error prone than relying on metric names (that change relatively often, so we'd break this and only notice much later).

Reply:
Done.